cookiecutter.json

```diff
@@ -2,16 +2,14 @@
   "author": "<INSERT AUTHOR HERE>",
   "author_email": "<INSERT EMAIL HERE>",
   "github_url": "If you have already made a github repo to tie the project to place it here, otherwise update in setup.py later.",
-  "adapter_name": "My Adapter",
-  "directory_name": "{{ cookiecutter.adapter_name.lower().replace(' ','') }}",
-  "project_name": "dbt-{{ cookiecutter.directory_name }}",
-  "description": "the {{ cookiecutter.directory_name }} adapter plugin for dbt.",
+  "adapter_name": "What is name of your database in CamelCase? ex. MyAdapter",
```

Comment:
Any reason why camelcase is important here but then we use the lowercase name for the project name?

Reply:
project name is used for the outermost level of the cookiecutter (specific need) generated project the overall folder name whereas directory_name which is now made in the hooks is used for rest of directories, and adapter_name is tagged into classnames where we would write them in camelcase and is more inline with how we might ref just the database naming in docs e.g BigQuery